display-servers/xlib-display-server/src/xwrap/getters.rs

```diff
@@ -638,11 +635,9 @@ impl XWrap {
     /// Returns all the roots of the display.
     // `XRootWindowOfScreen`: https://tronche.com/gui/x/xlib/display/screen-information.html#RootWindowOfScreen
     #[must_use]
-    fn get_roots(&self) -> Vec<xlib::Window> {
+    fn get_roots(&self) -> impl Iterator<Item = xlib::Window> + '_ {
```

Comment:
Only this one thing: Please remove the `#[must_use]` on the functions to fix the clippy warning, thanks. Otherwise LGTM.

Reply:
I saw this clippy as well on the GitHub diff view, I guess GH is doing those checks with the version that's in the toolchain and not what is set in the CI, so it doesn't come up in checks.

There is also one further clippy lint out, that is not related to this PR, so maybe it's time for a chores PR.